src/poliastro/integrator_params.py

```diff
@@ -0,0 +1,100 @@
+import numpy as np
+
+A = []
```

Comment:
Could we do

```
A = [
    np.array([5.26001519587677318785587544488e-2]),
    np.array([1.97250569845378994544595329183e-2, 5.91751709536136983633785987549e-2]),
    ...
]
```

instead? Same for the rest of the lists.

Also, is there a reason they have to be arrays? Can we just do

```
A = [
    [5.26001519587677318785587544488e-2],
    [1.97250569845378994544595329183e-2, 5.91751709536136983633785987549e-2],
    [2.95875854768068491816892993775e-2, 0, 8.87627564304205475450678981324e-2]
    ...
]
```

?